spec/git-repository-provider-spec.coffee

```diff
@@ -56,6 +56,21 @@ describe "GitRepositoryProvider", ->
           provider.repositoryForDirectory(directory).then (result) ->
             expect(result).toBe null
 
+    describe "when specified a Directory with a valid gitfile-linked repository", ->
+      it "returns a Promise that resolves to a GitRepository", ->
+        waitsForPromise ->
+          provider = new GitRepositoryProvider atom.project
+          gitDirPath = path.join(__dirname, 'fixtures/git/master.git')
```

Comment:
@maxbrunsfeld I expect you are correct.  But I copied that line from a previous test (line 27).